### DISCUSSION TITLE
lightweight dendron or alternative?

### DISCUSSION THREAD

**Discussion by Ding-Fan** [dendronhq/dendron#3302] [General]
My computer freeze frequently when I open vscode+dendron . Looks like the memory is not enough for it. Is there any option to keep dendron less resource demanding or a lightweight alternative? I wish I could migrate to a (neo)vim plugin.

**Comment by levirs565** [comment]
You can use [Obsidian](https://obsidian.md/) and [Dendron Tree Plugin for Obsidian](https://github.com/levirs565/obsidian-dendron-tree).

![Dendron Tree](https://github.com/levirs565/obsidian-dendron-tree/raw/master/images/dendron-tree.png)

Dendron Tree plugin will add tree to explore Dendron note. This plugin also add custom renderer for ref/embed and custom resolve for link. This plugin suport Obsidian in dekstop and mobile platform. You can install this plugin from [Obsidian plugin repository](https://obsidian.md/plugins?search=dendron)

**Reply by TomLucidor** [reply]
Have you tried Obsidian Copilot or Obsidian YOLO along with it?

**Comment by crwatanabe** [comment]
The biggest reason Obsidian isn't a replacement for Dendron is that it's an editor :pleading_face:
